src/main/scala/com/redislabs/provider/redis/redisFunctions.scala

```diff
@@ -359,6 +371,23 @@ object RedisContext extends Serializable {
     conn.close()
   }
 
+  /**
+    * @param geoName
+    * @param arr k/vs which should be saved in the target host
+    *            save all the k/vs to zsetName(zset type) to the target host
```

Comment:
This looks to be wrong. Copy pasted from another method?

Reply:
@hyungryuk ?